REDHAT.rst

```diff
@@ -0,0 +1,45 @@
+.. |br| raw:: html
```

Comment:
Not sure if root is the correct place to place this rst file, I can see RESPBIAN.rst file there too which should also be removed. All the .rst files should be placed inside docs and their entry point should be defined in docs/index.rst. This is how docs will be compile and build for readthedocs.

Reply:
done as suggested.